### PR TITLE
add rebalance heart beat

### PR DESCRIPTION
**Pull request by tyluffy**

### CODE REVIEW

File: pkg/kafsar/group_coordinator_impl_test.go [hunk 1]
```diff
@@ -138,3 +138,14 @@ func TestLeaveGroup(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, service.NONE, leaveGroupResp.ErrorCode)
 }
+
+func TestHeartBeat(t *testing.T) {
```
Comment (Shoothzj) [inline pkg/kafsar/group_coordinator_impl_test.go:142]: could you please add test for groupId empty?

Comment (goflutterjava) [inline pkg/kafsar/group_coordinator_impl_test.go:142]:
```suggestion
func TestHeartBeatRebalanceInProgress(t *testing.T) {
```

**Review by Shoothzj — APPROVED** [review]
